scripts/zombienet/local-testnet.toml

```
@@ -1,5 +1,5 @@
 [relaychain]
-default_command = "../polkadot-sdk/target/release/polkadot"
+default_command = "../../polkadot-sdk/target/release/polkadot"
```

Rearrange your local folders to fit the previous directory path
```suggestion
default_command = "../polkadot-sdk/target/release/polkadot"
```